src/ucs/debug/log.c

```diff
@@ -331,3 +331,21 @@ void ucs_log_cleanup()
     ucs_log_initialized    = 0;
     ucs_log_handlers_count = 0;
 }
+
+void ucs_log_print_backtrace(ucs_log_level_t level)
```

Comment:
pls add some test which calls this function

Reply:
added gtest